src/test/java/proj/kedabra/billsnap/business/service/impl/ItemServiceImplTest.java

```diff
@@ -0,0 +1,144 @@
+package proj.kedabra.billsnap.business.service.impl;
+
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.doAnswer;
+import static org.mockito.Mockito.when;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.Optional;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import proj.kedabra.billsnap.business.dto.ItemDTO;
+import proj.kedabra.billsnap.business.exception.ResourceNotFoundException;
+import proj.kedabra.billsnap.business.mapper.ItemMapper;
+import proj.kedabra.billsnap.business.model.entities.Item;
+import proj.kedabra.billsnap.business.repository.ItemRepository;
+import proj.kedabra.billsnap.fixtures.AccountEntityFixture;
+import proj.kedabra.billsnap.fixtures.BillEntityFixture;
+import proj.kedabra.billsnap.fixtures.EditBillDTOFixture;
+import proj.kedabra.billsnap.fixtures.ItemEntityFixture;
+import proj.kedabra.billsnap.utils.ErrorMessageEnum;
+
+public class ItemServiceImplTest {
+
+    @Mock
+    private ItemRepository itemRepository;
+
+    @Mock
+    private ItemMapper itemMapper;
+
+    private ItemServiceImpl itemService;
+
+    @BeforeEach
+    void setUp() {
+        MockitoAnnotations.initMocks(this);
+        itemService = new ItemServiceImpl(itemRepository, itemMapper);
+    }
```

Comment:
we can replace these types of `setUp()` with InjectMocks on class under test, and @ExtendWith(MockitoExtension.class) on test class

Reply:
o hey I didn't know we even added that class in, good to know